pom.xml

```diff
@@ -29,7 +29,7 @@
         <!-- JaCoCo needs this as a property -->
         <argLine>-Duser.language=en -Xmx1024m</argLine>
         <generateBackupPoms>false</generateBackupPoms>
-        <spring-boot.version>1.5.9.RELEASE</spring-boot.version>
+        <spring-boot.version>2.0.0.RELEASE</spring-boot.version>
```

Comment:
2.0.3.RELEASE is already out. Maybe we should use the latest available here?

Reply:
I don't think a library should force the use of the latest version of its dependencies, unless it doesn't work without a specific version. Liquigraph is not the Spring Boot BOM. 😅

Reply:
Fair enough. I'm always undecided which version to use when developing such library and decided to got with `latest` because it removes the case somebody looks into my POM and uses that version within his own app. But I'm fine with your explanation as well.